src/app/components/Slideshow/sliderComponents/Indicators.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const SliderIndicator = props => {
+  const { index, onClick, activeIndex } = props;
+
+  return (
+    <li>
+      <button
+        className={index === activeIndex ? 'slider__indicator slider__indicator--active' : 'slider__indicator'}
```

Comment:
`slider__indicator` shouldn't be used in condition

Reply:
why? changing styles is based on active index

Reply:
move ternary operator to variable